documentation/book/assembly-customizing-deployments.adoc

```diff
@@ -0,0 +1,18 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='assembly-customizing-deployments-{context}']
+= Customizing deployments
```

Comment:
@ppatierno . Can you think of a more descriptive title for the assembly? Perhaps something like:

## Customizing deployments using templates

Suggestions welcome!